modules/videoio/src/cap_gstreamer.cpp

```diff
@@ -924,6 +924,11 @@ bool GStreamerCapture::open(const String &filename_)
         gst_app_sink_set_max_buffers(GST_APP_SINK(sink.get()), 1);
     }
 
+    if (!manualpipeline)
+    {
+        gst_base_sink_set_sync(GST_BASE_SINK(sink.get()), FALSE);
```

Comment:
> v4l2src

`GStreamerCapture::open(int id)` provides "manual" pipeline, so its behavior is not changed

Reply:
Yes, but it is also possible to pass URI, e.g. `v4l2:///dev/video1` which will create automatic pipeline with [urisourcebin](https://gstreamer.freedesktop.org/documentation/playback/urisourcebin.html).